### PR TITLE
WIP: Bugfixes

### PR DESCRIPTION
**Pull request by CPatchane**
* Use fullwdith for the selectBox
* Remove some css warning about mixed value `start`
* ~~Fix the modal behavior on Safari/iOS which don't allow to use `vh`/`vw` in CSS `calc()`~~
* Fix background issue when opening modals in the app (using Portal)
* Get back permissions icons
screenshot 2019-01-24 at 15 06 18

### CODE REVIEW

**Comment by CPatchane** [comment]
[[STO] Reparer les icones des permissions](https://trello.com/c/8eCdWe6q/866-sto-reparer-les-icones-des-permissions)

File: src/styles/application.css [hunk 1]
```diff
@@ -99,7 +99,7 @@ main > [role=main].sto-myapps {
   }
 
   .sto-modal-page-app {
-    height: calc(100vh - 6rem)
+    height: calc(100% - 6rem)
```
Comment (gregorylegarec) [inline src/styles/application.css]: I am pretty sure it will cause an regression in responsive mode.

Comment (CPatchane) [reply src/styles/application.css]: This was an issue actually. Safari and Safari iOS don't handle `vh` and `vw` in calc function, this is why we changed that here.

Comment (GoOz) [reply src/styles/application.css]: Actually not sure we need this change anymore, since we used Portal. Worth checking.

Comment (CPatchane) [reply src/styles/application.css]: This fis seems not necessary anymore, so I removed the commit

**Pull request merged**